aggregate-guide.rst

```diff
@@ -2,7 +2,7 @@
 ODK Aggregate
 ******************
 
-`ODK Aggregate` provides a ready-to-deploy server and data repository to:-
+`ODK Aggregate` provides a ready-to-deploy server and data repository to :-
```

Comment:
It is not standard to put a space before colons.

I *would* like to see the hyphens after commas removed, as that is non-standard.